src/test/java/seedu/foodrem/logic/parser/generalcommandparser/HelpCommandParserTest.java

```diff
@@ -112,101 +109,102 @@ void parse_generalCommands() {
-                getHelp(getCommandWordFromCommandWordString("dec")));
+                getCommandWordFromCommandWordString("dec").getHelp());
         assertEquals(DeleteCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("del")));
+                getCommandWordFromCommandWordString("del").getHelp());
         assertEquals(NewTagCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("newtag")));
+                getCommandWordFromCommandWordString("newtag").getHelp());
         assertEquals(RenameTagCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("renametag")));
+                getCommandWordFromCommandWordString("renametag").getHelp());
         assertEquals(ResetCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("reset")));
+                getCommandWordFromCommandWordString("reset").getHelp());
         assertEquals(FindCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("find")));
+                getCommandWordFromCommandWordString("find").getHelp());
         assertEquals(ListCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("list")));
+                getCommandWordFromCommandWordString("list").getHelp());
         assertEquals(SortCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("sort")));
+                getCommandWordFromCommandWordString("sort").getHelp());
         assertEquals(ExitCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("exit")));
+                getCommandWordFromCommandWordString("exit").getHelp());
         assertEquals(HelpCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("help")));
+                getCommandWordFromCommandWordString("help").getHelp());
         assertEquals(TagCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("tag")));
+                getCommandWordFromCommandWordString("tag").getHelp());
         assertEquals(UntagCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("untag")));
+                getCommandWordFromCommandWordString("untag").getHelp());
         assertEquals(DeleteTagCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("deletetag")));
+                getCommandWordFromCommandWordString("deletetag").getHelp());
         assertEquals(ListTagCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("listtag")));
+                getCommandWordFromCommandWordString("listtag").getHelp());
     }
 
     @Test
     void getHelp_throws_error() {
-        assertThrows(ParseException.class, () -> getHelp(getCommandWordFromCommandWordString("helps")));
+        assertEquals(DEFAULT.getHelp(),
+                getCommandWordFromCommandWordString("helps").getHelp());
```

Comment:
What was this change about?

Reply:
On a second look, I can see why this change was made. In that case, should we combine this with the `get_help_success` method?

Reply:
Ah thank you for pointing this out. I did not realise it when changing.